lib/chef/resource/aws_rds_subnet_group.rb

```diff
@@ -0,0 +1,20 @@
+require 'chef/provisioning/aws_driver/aws_resource'
+require 'chef/resource/aws_subnet'
+
+class Chef::Resource::AwsRdsSubnetGroup < Chef::Provisioning::AWSDriver::AWSResource
+  aws_sdk_type AWS::RDS
+
+  attribute :db_subnet_group_name, kind_of: String, name_attribute: true
+  attribute :db_subnet_group_description, kind_of: String
```

Comment:
These are mapping directly to the required args that are used to create the subnet group:

http://docs.aws.amazon.com/AWSRubySDK/latest/AWS/RDS/Client.html#create_db_subnet_group-instance_method